### ISSUE TITLE
Pose estimation model with different number of keypoints

### ISSUE THREAD

**Issue by edwardnguyen1705** [yuanc3/Pose2ID#11]
Dear @yuanc3 ,

Thanks for sharing your great work.

I see that

`We use [DWpose](https://github.com/IDEA-Research/DWPose) to get poses with 18 keypoints.Please follow their official instructions. You may also use other pose estimation methods to get the 18 keypoints poses.`

Could I use a pose estimate model, but it outputs 17 (or different number) keypoints?

**Comment by yuanc3** [comment]
I haven’t tried it, but it may could. The training data may miss some points, so it can handle this situation. (Also some examples in the random generation Figure in our paper have missing keypoints)
Just note that keeping the colors consistent with the traditional keypoints map, since it will decide the direction of pedestrians.

**Comment by edwardnguyen1705** [comment]
> I haven’t tried it, but it may could. The training data may miss some points, so it can handle this situation. (Also some examples in the random generation Figure in our paper have missing keypoints) Just note that keeping the colors consistent with the traditional keypoints map, since it will decide the direction of pedestrians.

Thanks for the detailed information.

As I understand, if I use your `pretrained IPG models` and `standard_poses` to generate `#poses` images for each image of each personId in either gallery or query, then use techniques in `Quick Start` to enhance/centralize visual features (#features = k = embedding dim) to improve ReID/matching results. If so, I do not need obtain target pose of each ref image?

**Comment by edwardnguyen1705** [comment]
> Yes, it does not need obtain pose of ref image. And, actually, you can use as more poses as you can to generate more images and fuse them together for better results.

Thank you!
